.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+### Description
+
+    Describe your changes
+
+### Jira Ticket
+
+    i.e https://mozilla-hub.atlassian.net/browse/XXX-####
```

Comment:
We cannot ask this for community contributions. Say Jira or Github, for community contributions.